src/lib/features/feature-search/feature.search.e2e.test.ts

```diff
@@ -1252,3 +1252,39 @@ test('should return archived when query param set', async () => {
         ],
     });
 });
+
+test('should return tags with color information from tag type', async () => {
+    await app.createFeature('my_feature_a');
+
+    await app.request
+        .put('/api/admin/tag-types/simple')
+        .send({
+            name: 'simple',
+            description: 'Simple tag type',
```

Comment:
what are we communicating by including this description in this test? I'd omit this if this is not relevant or call it 'irrelevant'